openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -275,6 +275,7 @@
     "data-insight-chart": "Data insight chart",
     "data-assets": "Data Assets",
     "app-analytics": "App Analytics",
+    "no-team-found": "No team found with name given.",
```

Comment:
```suggestion
    "no-team-found": "No team found with the {{name}}.",
```

Reply:
Changing it to generic as "No team found" as we might not name with us